xrm/index.d.ts

```diff
@@ -394,11 +605,27 @@ declare namespace Xrm
     export interface Ui
     {
         /**
+          * Displays a form level notification. Any number of notifications can be displayed and will remain until removed using clearFormNotification.
+          * The height of the notification area is limited so each new message will be added to the top.
+          *
+          * @param   {string}                            message     The text of the notification message.
+          * @param   {Page.ui.FormNotificationLevel}     level    The level of the notification which defines how the message will be displayed, such as the icon.
+          * @param   {string}                            uniqueId    Unique identifier for the notification which is used with clearFormNotification to remove the notification.
+          *
+          * @return  true if it succeeds, otherwise false.
+          *
+          * @remarks     Values for level are:    ERROR   Notification will use the system error icon.
+          *                                       WARNING Notification will use the system warning icon.
+          *                                       INFO    Notification will use the system info icon.
+          */
+        setFormNotification( message: string, level: Page.ui.FormNotificationLevel | string, uniqueId: string ): boolean;
```

Comment:
Thinking we should remove the union type string   ERROR, WARNING and INFO are the only options.